src/main/java/converter/StringToUserConverter.java

```diff
@@ -6,8 +6,9 @@ public class StringToUserConverter {
 
     public User convert(String line) {
 
+
         if (line == null || line.isBlank()) {
-            throw new IllegalArgumentException("String = null or Blank");
+            throw new IllegalArgumentException("line = null");
```

Comment:
You throw an exception when line is null OR empty. 
String.format(“Illegal argument line = [%]”, line)

Reply:
And check other messages